main.py

```diff
@@ -0,0 +1,22 @@
+from flask import Blueprint, current_app, request
+from membrane.client.flask import membrane_current_user, membrane_login_required
+
+main = Blueprint("main", __name__)
```

Comment:
__name__ evaluates to the module. there's no module at the root level right? shouldn't this be placed in a module testapp/__init__.py to haev a reasonable name? 

main.py is a terrible name.

Reply:
Done. Moved to routes/init.py.